### PR TITLE
feat: update swc plugin template

### PR DESCRIPTION
**Pull request by magic-akari**
**Description:**

We should add the swc plugin template to the test. This will prevent forgetting to update the template when the plugin is updated.

### CODE REVIEW

**Review by kdy1 — CHANGES_REQUESTED** [review]
Can you fix the benchmark action?
Seems like there's an issue with cargo feature unification.

I think `swc_plugin_testing` should be moved to `bindings` workspace

File: bindings/swc_cli/src/commands/plugin.rs [hunk 1]
```diff
@@ -231,55 +238,7 @@ build-wasm32 = "build --target wasm32-unknown-unknown"
...
-    Default::default(),
-    |_| as_folder(TransformVisitor),
-    boo,
-    // Input codes
-    r#"console.log("transform");"#,
-    // Output codes after transformed with plugin
-    r#"console.log("transform");"#
-);"##
-                .as_bytes(),
+            include_bytes!("../../../../crates/swc_plugin_testing/src/template.rs"),
```
Comment (kdy1) [inline bindings/swc_cli/src/commands/plugin.rs:241]: This will not work if this crate is published

Comment (magic-akari) [reply bindings/swc_cli/src/commands/plugin.rs:241]: You are right. I forgot that crate can be installed and used separately.

**Comment by magic-akari** [comment]
I did not managed to move it without build error.

**Pull request closed without merging**